src/AspNet.Security.OAuth.Apple/AppleAuthenticationHandler.cs

```diff
@@ -342,17 +342,6 @@ protected override async Task<HandleRequestResult> HandleRemoteAuthenticateAsync
             properties.StoreTokens(authTokens);
         }
 
-        if (parameters.TryGetValue("user", out var userJson))
-        {
-            using var user = JsonDocument.Parse(userJson);
-            var userClaims = ExtractClaimsFromUser(user.RootElement);
-
-            foreach (var claim in userClaims)
-            {
-                identity.AddClaim(claim);
-            }
-        }
-
```

Comment:
This breaks things in the case of the form post flow (the default) where the parameters aren't in the query string.

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/4b6591108b5d576005a7115e244b28f34339183c/src/AspNet.Security.OAuth.Apple/AppleAuthenticationHandler.cs#L177-L181

Reply:
I think the proper way to remove this code would be to have the override just call the base method, and the duplicated implementation that supports query and form can just be deleted.

Reply:
it is still valid, as it ignores the user field from the form post deliberately.

Reply:
Indeed, remove the boilerplate code and reuse the base would be a good idea. As long as the apple specific design be removed.